src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseUnlinkedNotificationTest.java

```diff
@@ -48,10 +48,14 @@ void shouldNotifySubjectOfApplicationReceivedWithEmail() {
 
         //Then
         verify(notificationService).sendEmail(any(NotificationRequest.class));
+        verify(notificationHelper).buildEmailNotificationRequest(
```

Comment:
Please update this test name. Needs updating since it does not notify that the application has been received.